README.md

```diff
@@ -78,38 +97,43 @@ If you're using Windows, you can still download and use bash (maybe try using gi
 If you're stuck with Powershell... helaas pindakaas...
 
 ## Monitoring
+
 ### Prometheus
 
 To access Prometheus UI and Query metrics you can port forward from a new terminal in `./operation`:
+
 ```zsh
 ./scripts/update-hosts.sh
 kubectl port-forward -n monitoring svc/myprom-kube-prometheus-sta-prometheus 9090
```

Comment:
Note to myself: update README, no more port-forwarding is needed and prometheus can be accessed by URL.